examples/adminapi.py

```diff
@@ -274,6 +274,19 @@ def example_list(a, args):
                   " isrs: {} errstr: {}".format(p.id, p.leader, p.replicas,
                                                 p.isrs, errstr))
 
+    groups = a.list_groups(timeout=10)
```

Comment:
Add two specific command options and dont include them in this metadata listing.
 - ` .. list groups`
 - ` .. list group <group-id>`